src/main/resources/org/jenkinsci/plugins/electricflow/ElectricFlowDeployApplication/config.jelly

```diff
@@ -117,20 +117,40 @@ All rights reserved.
                     }
 
 
-                    var row1 = table.insertRow(table.rows.length - 2);
-                    row1.className = "_ef_da_row";
-                    if (parameters.length) {
-                        row1.appendChild(createTitleForDeployApplication("Deploy Parameters"));
-                        parameters.forEach(function (elem) {
-                            var row = table.insertRow(table.rows.length - 2);
-                            row.className = "_ef_da_parameters _ef_da_row";
-                            addParameterRowForDeployApplication(row, elem.actualParameterName, elem.value, false);
-                        });
+                    if (table == null) {
```

Comment:
Where does this variable come from?

Reply:
this was used in older version of the code (and still to be used for older versions of Jenkins). See the code above:
```
            var table = document.querySelector("[descriptorid='electricFlowDeployApplication'] table");
            if (table == null) {
                table = getElementByXpath("//select[@checkdependson='validationTrigger']/ancestor::table[@name]");
            }
```

Reply:
this was used in older version of the code (and still to be used for older versions of Jenkins). See the code above:
```
            var table = document.querySelector("[descriptorid='electricFlowDeployApplication'] table");
            if (table == null) {
                table = getElementByXpath("//select[@checkdependson='validationTrigger']/ancestor::table[@name]");
            }
```